rm-services.yml

```
@@ -289,6 +289,21 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
+  support-frontend:
+    container_name: support-frontend
+    image: europe-west2-docker.pkg.dev/ssdc-rm-ci/docker/srm-support-frontend:latest
+    environment:
+      - APP_CONFIG=DevelopmentConfig
+      - PORT=${SUPPORT_FRONTEND_PORT}
+    ports:
+      - "${SUPPORT_FRONTEND_PORT}:9096"
+    healthcheck:
+      test: [ "CMD", "python", "-c", "import urllib.request as r; r.urlopen('http://localhost:9096/info')" ]
```

```suggestion
      test: [ "CMD", "python", "-c", "import urllib.request as r; r.urlopen('http://localhost:9096/info/')" ]
```
Worth adding the trailing slash. It automatically redirects but it's probably worth adding it